pyproject.toml

```diff
@@ -46,7 +46,19 @@ documentation = "https://ietf-tools.github.io/xml2rfc/"
 issues = "https://github.com/ietf-tools/xml2rfc/issues"
 
 [project.optional-dependencies]
-pdf = ["weasyprint==65.0"]
+pdf = [
+    "weasyprint==65.0",
+]
+tests = [
+    "certifi",
+    "decorator",
+    "dict2xml",
+    "pyflakes",
+    "pypdf>=4.1.0",
+    "tox",
+    "tox-gh-actions",
+    "weasyprint==65.0",
```

Comment:
Sorry for not spotting this earlier, This includes bunch of test dependencies that's only specific to github actions.
The list should only have:
```
    pyflakes
    decorator
    dict2xml
    pypdf>=4.1.0
    weasyprint==64.1
    tox
```